### PR TITLE
Docs update

### PR DESCRIPTION
**Pull request by rosteen**
Updating the docs to be a little more user friendly.

### CODE REVIEW

**Review by kecnry — COMMENTED** [review]
I like this move... but should we put the new rst file somewhere in the already existing docs/dev (and link to it from docs/dev/index)?

File: docs/developer.rst [hunk 1]
```diff
@@ -0,0 +1,16 @@
+
+.. _developer:
+
+Reference Files
+---------------
+The HST reference files in config/hst may occasionally need to be remade with
```
Comment (kecnry) [inline docs/developer.rst:6]:
```suggestion
The HST reference files in ``config/hst`` may occasionally need to be remade with
```

**Review by kecnry — APPROVED** [review]
Few minor styling tweak suggestions, otherwise looks good and a definite improvement to me!

File: docs/developer.rst [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+updated values from the instrument teams. 
+A convenience generator for the reference files is provided in 
+`astrogrism.HST.create_HST_reffiles.py`, which creates the specws and wavelengthrange
+reference files based on a GRISMCONF configuration file. The distortion generator 
+is currently in progress. To generate the files, provide the path to the GRISMCONF 
+configuration file and the name of the grism as arguments to the 
+`create_reference_files` function::
```
[the remark this answers is not in the document]
Comment (rosteen) [reply docs/developer.rst:13]: I don't think so, unfortunately, only the `GrismObs` base class shows up in the docs. Future improvement.

**Pull request merged**